exercises/hangman/example.py

```diff
@@ -0,0 +1,42 @@
+class Hangman:
+    def __init__(self, word):
+        self.remainingGuesses = 9
+        self.status = 'busy'
```

Comment:
I think `"busy"` is a little confusing as a status - it makes me think that the class is currently doing something and can't respond. `"in progress"` might be better?

A better approach to handling status would be to declare some constants at the start of the class (`WIN`, `LOSE`, `UNFINISHED`). The class would then set with `self.status = self.WIN` and tested with `self.assertEqual(game.status, game.WIN)`. This approach frees up the learner to chose whatever values they find most appropriate/convenient for handling game status.

I also think that we need a test to check that a game that has been lost refuses further interaction. Currently, your example would fail this test, since if I just guess every letter in the alphabet, the status would go to `"lose"` and then back to `"win"` once I have guessed all of the letters in the word.

Reply:
How do want to stop the game after it is finished (either win or lose)? Should I raise an Exception or Error? or should i just use an if statement and stop it from updating everything in guess()?

Reply:
@khoa102 Can you clarify what you mean by "stop the game"? Since this is not an interactive program, "stop" loses its meaning.

Reply:
@cmccandless N-Parson mentions earlier that "a game that has been lost refuses further interaction." And that is what I mean by stopping the game after it's finished (win or lose)